x-pack/plugin/blob-cache/src/main/java/org/elasticsearch/blobcache/shared/SharedBlobCacheService.java

```diff
@@ -253,6 +253,14 @@ public void validate(ByteSizeValue value, Map<Setting<?>, Object> settings, bool
         Setting.Property.NodeScope
     );
 
+    public static final Setting<Boolean> USE_FULL_REGION_SIZE = Setting.boolSetting(
+        SHARED_CACHE_SETTINGS_PREFIX + "use_full_region_size",
```

Comment:
Nit: I feel this is more of `always_use_full_region_size` since we do already use full region size sometimes.

Reply:
Marking comment as resolved since this is now obsolete